src/Sarif/ToDotNet/ToDotNet.targets

```diff
@@ -57,12 +52,5 @@
 
     <Copy SourceFiles="@(_NotYetGeneratedFiles)"		
             DestinationFolder="$(_ObjectModelOutputDirectory)" />
-
-    <ItemGroup>
-      <!-- FileWrites are the set of files that get deleted on clean. -->
-      <FileWrites Include="$(_ObjectModelOutputDirectory)\*.cs" />
-      <!-- Compile are the set of files that go into the C# build. -->
-      <Compile Include="$(_ObjectModelOutputDirectory)\*.cs" />
-    </ItemGroup>
```

Comment:
Again, since we no longer build this target in the course of the compilation, there's no need to dynamically add `Compile` items. It was that dynamic addition of `Compile` items that wasn't working properly with the new-style `.csproj` files.

Reply:
This is interesting  bit of information to know.  Good catch.

---
In reply to: [147594338](https://github.com/Microsoft/sarif-sdk/pull/660#discussion_r147594338) [](ancestors = 147594338)